lib/Dialect/Substrait/IR/Substrait.cpp

```diff
@@ -68,6 +68,19 @@ LogicalResult AdvancedExtensionAttr::verify(
   return success();
 }
 
+LogicalResult mlir::substrait::FixedCharAttr::verify(
+    llvm::function_ref<mlir::InFlightDiagnostic()> emitError, StringAttr value,
+    Type type) {
+  FixedCharType fixedCharType = mlir::dyn_cast<FixedCharType>(type);
+  if (fixedCharType == nullptr)
+    return emitError() << "expected a fixed char type";
```

Comment:
Nit: use the type name from the assembly format and give details on where it was expected.
```suggestion
    return emitError() << "expected 'type' to be a 'fixed_char'";
```